OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -203,7 +203,13 @@ public void navigateNextComponent() {
         if (index < pagerAdapter.getItemCount() - 1) {
             pager2.setCurrentItem(index + 1);
         }
-        if (selectedUnit.isLastChild() && isFirstSection && !isVideoMode) {
+        // CourseComponent#getAncestor(2) will return the section of the xBlock
```

Comment:
CourseComponent#getAncestor(2) returns the section of a component

Reply:
done.